data-service/templates/data_service.yml

```diff
@@ -2,60 +2,68 @@ AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
 Resources:
 
-  EnvironmentsTable:
+  EnvironmentTable:
     Type: AWS::DynamoDB::Table
     Properties:
      #TODO: change ddb table names to have region prefix passed in as a resource
      TableName: Environments
      AttributeDefinitions:
        - AttributeName: environmentId
          AttributeType: S
-       - AttributeName: environmentVersion
+       - AttributeName: environmentName
+         AttributeType: S
+       - AttributeName: accountId
+         AttributeType: S
+       - AttributeName: cluster
          AttributeType: S
      KeySchema:
        - AttributeName: "environmentId"
          KeyType: HASH
-       - AttributeName: "environmentVersion"
+       - AttributeName: "environmentName"
          KeyType: RANGE
      ProvisionedThroughput:
        ReadCapacityUnits: 15
        WriteCapacityUnits: 15
+     GlobalSecondaryIndexes:
+       -
+         IndexName: "environment-cluster-index"
+         KeySchema:
+           - AttributeName: "accountId"
+             KeyType: HASH
+           - AttributeName: "cluster"
+             KeyType: RANGE
+         Projection:
+           ProjectionType: "KEYS_ONLY"
+         ProvisionedThroughput:
+           ReadCapacityUnits: 15
+           WriteCapacityUnits: 15
 
-  EnvironmentTargetVersionTable:
+  EnvironmentRevisionTable:
     Type: AWS::DynamoDB::Table
     Properties:
      #TODO: change ddb table names to have region prefix passed in as a resource
-      TableName: EnvironmentTargetVersion
+      TableName: EnvironmentRevision
```

Comment:
Should this be `EnvironmentRevisions` to match the plural `Environments` table?

Reply:
sure